guillotina/json/deserialize_content.py

```diff
@@ -117,11 +117,14 @@ async def set_schema(
             else:
                 data_value = data[name] if name in data else None
                 found = True if name in data else False
-            if found:
 
-                if not self.check_permission(write_permissions.get(name)):
+            if found or field.missing_value is not None:
+                if found and not self.check_permission(write_permissions.get(name)):
                     raise Unauthorized("Write permission not allowed")
 
+                if field.missing_value is not None:
```

Comment:
This will not set the value always to the missing_value if its defined?

Reply:
ups, yes